backend-lambda/index.js

```diff
@@ -11,8 +12,13 @@ const headers = {
 export const handler = async (event, context) => {
   const body =
     typeof event.body === "string" ? JSON.parse(event.body) : event.body;
+  // define path string to check for product_id in updateProduct
+  const path = event.path;
+  const httpMethod = event.httpMethod;
   console.info("event", event);
   console.info("context", context);
+  // Log the parsed body in a formatted way
+  console.info(body);
```

Comment:
This is a bit redundant since it's in the event log on L#18 but we can keep it.

Reply:
Removed